packages/github/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.18.0"
+  changes:
+    - description: Update unique keys for latest issues transform
+      type: bugfix
```

```suggestion
- version: "2.17.1"
  changes:
    - description: Update unique keys for latest issues transform.
      type: bugfix
```
Bugfixes bump patch, but you've noted in the description and in the transform version that this is breaking. ISTM that this should be reflected in the package version as well; if a subcomponent has a breaking change then the component has a breaking change, unless the component hides it, which I do not believe is the case here.